kubernetes/pods.json

```diff
@@ -52,7 +52,7 @@
     },
     {
       "property": "host",
-      "displayName": "Node"
+      "displayName": "Host"
```

Comment:
Seems like "Node" was the more natural Kubernetes term for that is the word used in K8s docs/cli tools.